scripts/build-tree-sitter-wasm.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Inspired by bash-language-server under MIT license
+# Reference: https://github.com/bash-lsp/bash-language-server/blob/8c42218c77a9451b308839f9a754abde901323d5/scripts/upgrade-tree-sitter.sh
+
+set -euox pipefail
+
+update-tree-sitter-wasm() {
```

Comment:
I'm wondering if we were using git submodules if someone hasn't already written an action to update them.
It would be better than maintaining our own fetch/update solutions.
See https://stackoverflow.com/a/73229032/20690788
I'm going to log this in a ticket

Reply:
this is an other option